PurchasesCoreSwift/Public/Error Handling/ErrorCode.swift

```diff
@@ -45,6 +45,7 @@ import Foundation
     @objc(RCLogOutAnonymousUserError) case logOutAnonymousUserError = 22
     @objc(RCConfigurationError) case configurationError = 23
     @objc(RCEmptySubscriberAttributesError) case emptySubscriberAttributes = 24
+    @objc(RCmissingAppUserIDForAliasCreationError) case missingAppUserIDForAliasCreationError = 26
```

Comment:
I did 26 because this is already queued up:
https://github.com/RevenueCat/purchases-ios/pull/733/files#diff-095f5b4949ca851714fae309e4a2fcbe59ac52fda5edd148a758f15a2343fd1aR48